Makefile

```diff
@@ -1,7 +1,7 @@
 OS := $(shell uname | awk '{print tolower($$0)}')
 MACHINE := $(shell uname -m)
 
-DEV_ROCKS = "busted 2.0.0" "luacheck 0.23.0" "lua-llthreads2 0.1.5" "http 0.3"
+DEV_ROCKS = "busted 2.0.0" "busted-htest 1.0.0" "luacheck 0.23.0" "lua-llthreads2 0.1.5" "http 0.3"
```

Comment:
not sure this should be in the make file. Since it's a preference, not required for testing, maybe simply add the installation to the `.travis.yml` ?

Reply:
The scripts assume gtest, which is a preference as well.

Reply:
To expand on this point: the `.ci/` scripts which run the tests specify the Busted output handler, and those depend on the tools built by the Makefile. They are designed to be CI-agnostic (and in fact, we're testing and about to migrate to Github Actions). If this moves to the `.travis.yml` file this produces a cross-dependency between the CI config file and the test scripts, which break their CI-agnosticity.

Reply:
I don't get your second point. My thinking is that the gtest one is bundled with busted. Busted is required anyway. So if this htest output is required in ci, then that's where it should be configured. 

Otherwise we'll start adding more "preferred" utilities to the makefile, which would clutter everybody's systems.

So imo, better stay away from that, keep the generic one lean and mean (makefile), and add the preferred output only to ci.

(But if we don't care, then I definitely have some fine utility functions to add to the test helpers for example)